thorlcr/activities/rollup/throllupslave.cpp

```diff
@@ -591,25 +591,29 @@ class CRollupGroupSlaveActivity : public CSlaveActivity, public CThorDataLink
     CATCH_NEXTROW()
     {
         ActivityTimer t(totalCycles, timeActivities);
-        if (!eoi)
+        if (eoi)
+            return NULL;
+
+        loop
         {
             CThorExpandingRowArray rows(*this, queryRowInterfaces(input));
```

Comment:
Likely to be when groups are small, though not new.
Could move out an call clearRows() instead, but will make that a separate PR.